internal/info/info.go

```diff
@@ -148,25 +209,25 @@ func (d Detail) String() string {
 }
 
 // Get returns parased Detail object.
-func (d Detail) Get() Detail {
-	d.prepare()
+func (i info) Get() Detail {
```

Comment:
is it pointer receiver? 🤔 

```suggestion
func (i *info) Get() Detail {
```

Reply:
I think keeping non-pointer receiver is better as we should not store stacktrace information to the receiver object, and a new instance of `Detail` can be returned from this function.